### PR TITLE
Add feeling safe question

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/52F4JfQd

Screenshot 2020-04-08 at 11 22 39
Screenshot 2020-04-08 at 11 22 48

### CODE REVIEW

File: spec/requests/feel_safe_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      it "shows the form with prefilled response" do
+        visit feel_safe_path
+
+        expect(page.body).to have_content(I18n.t("coronavirus_form.questions.feel_safe.title"))
+        expect(page.find("input#option_#{selected_option.parameterize.underscore}")).to be_checked
+      end
+    end
+  end
+
+  describe "POST /afford-food" do
```
Comment (issyl0) [inline spec/requests/feel_safe_spec.rb:30]:
```suggestion
  describe "POST /feel-safe" do
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -168,5 +168,10 @@ en:
         options:
           - "Yes"
           - "I might be evicted soon"
+      feel_safe:
+        title: "Do you feel safe where you live?"
+        options:
+          - "Yes"
+          - "Yes but I’m concerned about the safety of someone else"
```
Comment (brucebolt) [inline config/locales/en.yml:175]: You appear to have lost the No/Not Sure answers from the "Have you been evicted" question

File: spec/requests/feel_safe_spec.rb (hunk 1 above)
Comment (leenagupte) [reply spec/requests/feel_safe_spec.rb:30]: 🤦‍♂

File: config/locales/en.yml (hunk 2 above)
Comment (leenagupte) [reply config/locales/en.yml:175]: Ah. Must have happened during the rebase.

**Comment by leenagupte** [comment]
@issyl0 @brucebolt Thanks! I've updated and fixed-up.

**Pull request merged**